packages/lesswrong/lib/editor/make_editable.js

```diff
@@ -271,4 +272,15 @@ export const makeEditable = ({collection, options = {}}) => {
     //   }
     // },
   ])
+  if (pingbacks) {
+    addFieldsDict(collection, {
+      pingbacks: {
+        type: Object,
+        blackbox: true,
```

Comment:
what's "blackbox" about?

Reply:
It means you can throw whatever you want in there. This isn't great, and I would much prefer it to have a schema, but given that it's server-side write-only, it's not the end of the world. 

@jimrandomh: Do you want to add a schema to this?

Reply:
Added